Makefile

```diff
@@ -77,19 +88,19 @@ call_pyright:
 pyright: conda call_pyright
 
 # run basic checks, assuming test requirements already installed
-check: flake8 mypy
+check: flake8 mypy black
```

Comment:
shouldn't black run before flake8? Flake8 might detect errors that black will later remove

Reply:
In this context `black` doesn't make any changes as it is running with the flag `--diff --check`! so in theory the order shouldn't matter here